Plutarch/Pair.hs

```diff
@@ -18,3 +18,24 @@ data PPair (a :: PType) (b :: PType) (s :: S)
   deriving anyclass (PlutusType, PEq, PShow)
 
 instance DerivePlutusType (PPair a b) where type DPTStrat _ = PlutusTypeScott
+
+instance (PPartialOrd a, PPartialOrd b) => PPartialOrd (PPair a b) where
```

Comment:
Why not derive this

Reply:
I couldn't find infrastructure for deriving that. Is there any? 
Sorry, I haven't really had time to learn all possibilities of deriving-via yet.

Reply:
You can derive all classes that have a default for every method.

Reply:
The default needs an `POrd` instance for `PInner` of the type, which in the case of `PPair` is `PScottEncoded ...`. That one doesn't have a `POrd` instance. Are you sure this should be possible somehow?

Reply:
This is an issue. Look at the default methods. You can change the constraint to PPartialOrd, but what about deriving POrd? You can add the constraint as a superclass, but perhaps adding a dummy method with a POrd constraint on PInner works best.

Reply:
What mechanism should cause the lexical ordering behavior that you see in my manual implementation? I don't see how changing constraints would help with that. I suppose a better default implementation would have to use generics.